src/lib/condition.js

```diff
@@ -59,19 +61,34 @@ class Condition {
     }
 
     const pieces = serializedCondition.split(':')
-    if (pieces[0] !== 'cc') {
-      throw new PrefixError('Serialized condition must start with "cc:"')
+    if (pieces[0] !== 'ni') {
+      throw new PrefixError('Serialized condition must start with "ni:"')
     }
 
-    if (!Condition.REGEX_STRICT.exec(serializedCondition)) {
+    const parsed = Condition.REGEX_STRICT.exec(serializedCondition)
+    if (!parsed) {
       throw new ParseError('Invalid condition format')
     }
 
+    const query = require('querystring').parse(parsed[2])
```

Comment:
Oops, no, just a piece of hacky code that I missed during my 2nd pass. Will fix.